script/Ciel/Day 14/14_01B_01_CIEL14B_1D.txt

```diff
@@ -701,67 +784,73 @@ Ciel's maximum range is five kilometers.
 [sha:231d856d663db75dbaf1b7f2c109eb87323004a0]{
 -- Page 191, Offset 38885.
 -- それに比べれば500メートル先を狙うなぞ、彼女にとって針の穴に銃弾を通す<程|・><度|・>の難易度だ。
-Compared to that, aiming 500 meters away is the same difficulty as passing a bullet through an eye of a needle.
+// That still sounds hard?
+// Assuming this is supposed to be tongue in cheek and ・s over 程度 should make it render as 'easy'
+Compared to that, aiming at a target a mere five hundred meters away is as easy as firing a bullet through the eye of a needle.
 }
 [sha:6ad5f7185b5593f28c1f46a7cb65e08f3b4c4cd0]{
 -- Page 192, Offset 38886.
 -- 手袋越しの指が、<破城弩弓|ゆ　　み>の重さをグラム単位で推し量る。
-Using her fingers through her gloves, she estimates the bow's weight in grams.
+Through her gloves, she estimates the bow's weight down to the gram.
 }
 [sha:fdcb4b05e03ce7f0b4e8e70e157099db5d175784]{
 -- Page 192, Offset 38887.
 -- 風向きを測り、自身のコンディションを確認し、心臓のリズムを舌に乗せる。
-She measures the direction of the wind, checks her own condition, and records the rhythm of her heartbeat.
+// Assuming 舌に乗せる is eat -> stifle
+She measures the direction of the wind,
+ evaluates her own condition,
+ and swallows her heartbeat.
 }
 [sha:9c812b12d7831d275fdd2951bb40d331d2aa93c6]{
 -- Page 192, Offset 38888.
 -- 精緻極まる筋肉繊維の連続運動。
-A continuous movement of extremely delicate muscle fibers.
+Her muscle fibers continuously make minute adjustments to her aim.
 }
 [sha:d186b8569ed88791624f839cfe5d9d16dfc0d730]{
 -- Page 192, Offset 38889.
 -- “――――――今！”
-"――――――Now!"
+"―――Now!"
 }
 [sha:e3238692ef725f0164b13c98e04adcb1a888f7df]{
 -- Page 193, Offset 38890.
 -- 果たして、奇蹟はここに執り行われた。
-As a result, a miracle was performed.
+Sure enough, a miracle has been performed here.
 }
 [sha:4dccf34c1140524f3831e61fb519f74263a5f2b4]{
 -- Page 193, Offset 38891.
 -- “弓”の名を冠するもの。
-The one crowned as the 'bow'.
+// Flopped with below for EN flow
+This is the essence of the Executor spoken of as 'special' even within the den of monsters known as the Burial Agency―――
 }
 [sha:198721a67ec2eb4c786490bd53aee92daba52d18]{
 -- Page 193, Offset 38892.
 -- 魔人どもの巣である埋葬機関においてさえ“特例”と言わしめた代行者、その神髄。
-Even among the den of monstrous beings that is the Burial Agency, she is considered to be an Executor that is 'special'.
+The one anointed with the codename 'bow'.
```

Comment:
codename sounds tonally completely wack.

epithet works best for me because latin, and I don't really think 'epithet' has THAT much of a negative connotation